Procfile

```diff
@@ -0,0 +1,4 @@
+# Run Rails & Webpack concurrently
+# Example file from webpack-rails gem
+rails: bundle exec rails server
```

Comment:
Follow up, I also tried to edit the webpack configs to run on 0.0.0.0 with no luck which makes this hard to test when running from a VM based environment (and thus accessing it via ip address or hostname).